docs/models.md

```diff
@@ -7,3 +7,74 @@ Teacher               | Used to store *teacher_name* and their *department*.
 TeacherSubjectMapping | Used to store the realationship between Subject and Teacher model. Eg. X teacher teach Y subject.
 Section               | It store the seciton, branch, year, shift and Teacher -----> Subject record of a particular class.
 Feedback	      | Used to store feedback of every teacher on the basis of the subject he/she teach.
+
+## Model's Field
+
+**1. Subject**
+
+Field_Name       | Description
+-----------------|--------------------------------------------------------------------
+**subject_name** | This field is used to store all subject name in the databse.
+**paper_id**     | This field is used to store paper id of each subject in the databse.
+
+**2. Teacher**
+
+Field_Name      | Description
+----------------|---------------------------------------------
+**teacher_name**| It store every teacher name in the database.
+**teacher_dept**| It store department name of each teacher in the database.
```

Comment:
It should be `stores` rather than `store` or better choice could be `holds` in both the fields.